src/generate-bindings.sh

```diff
@@ -2,10 +2,6 @@
 
 set -ex
 
-# TODO: update to next wit-bindgen release once
-# https://github.com/bytecodealliance/wit-bindgen/pull/1026 and
-# https://github.com/bytecodealliance/wit-bindgen/pull/1027 are merged and
-# released
-cargo install --locked --no-default-features --features csharp --git https://github.com/dicej/wit-bindgen --branch csharp-public-imports-tmp wit-bindgen-cli --root $(pwd)
+cargo install --locked --no-default-features --features csharp wit-bindgen-cli --root $(pwd)
```

Comment:
```suggestion
cargo install --locked --no-default-features --features csharp --version 0.30.0 wit-bindgen-cli --root $(pwd)
```

Let's make the version explicit so we're not surprised by changes in future releases.  We can always bump that version when we're ready.

Reply:
done